REQUIRE

```diff
@@ -6,3 +6,4 @@ SortingAlgorithms
 Reexport
 Compat 0.8.4
 FileIO 0.1.2
+Juno
```

Comment:
If the tests require a specific version of Juno to be able to pass, that should be set as the minimum required Juno version.